tests/client/amo/components/TestLanguagePicker.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+
+import LanguagePicker from 'amo/components/LanguagePicker';
+import { getFakeI18nInst } from 'tests/client/helpers';
+import I18nProvider from 'core/i18n/Provider';
+
+
+describe('LanguagePicker', () => {
+  function renderLanguagePicker({ ...props }) {
+    return findRenderedComponentWithType(renderIntoDocument(
+      <I18nProvider i18n={getFakeI18nInst()}>
```

Comment:
True, but we've done things this way in the tests before. I like testing the component as it would be imported/used by the rest of the app.